scripts/localscript.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+mkdir -p /home/vscode/.local/bin
+
+chmod +x /workspaces/r-dev-env/scripts/rterm.sh
+
+mv /workspaces/r-dev-env/scripts/rpath.sh /home/vscode/.local/bin
```

Comment:
You have given the wrong file name here. Also, we use the opportunity to drop the file extension:

```suggestion
mv /workspaces/r-dev-env/scripts/rterm.sh /home/vscode/.local/bin/rterm
```

When you load the workspace make sure the script is in `~/.local/bin`, e.g. via `ls /home/vscode/.local/bin`. If it is then you should be able to run the script without giving the full path, i.e. just as `rterm`.